modular_zubbers/code/modules/jobs/job_types/nanotrasen_consultant.dm

```diff
@@ -19,3 +19,15 @@
 /obj/structure/closet/secure_closet/nanotrasen_consultant // Ditto of the previous for skyrat automapper placed lockers.
 	req_access = list()
 	req_one_access = list(ACCESS_CENT_GENERAL)
+
+/datum/id_trim/job/nanotrasen_consultant
+	minimal_access = list(
+				ACCESS_AI_UPLOAD, ACCESS_ALL_PERSONAL_LOCKERS, ACCESS_AUX_BASE, ACCESS_BAR, ACCESS_BRIG_ENTRANCE,
+				ACCESS_CENT_GENERAL, ACCESS_CHANGE_IDS, ACCESS_CHAPEL_OFFICE, ACCESS_COMMAND, ACCESS_CONSTRUCTION,
+				ACCESS_COURT, ACCESS_ENGINEERING, ACCESS_EVA, ACCESS_GATEWAY, ACCESS_HOP, ACCESS_HYDROPONICS,
+				ACCESS_JANITOR, ACCESS_KEYCARD_AUTH, ACCESS_SERVICE, ACCESS_KITCHEN, ACCESS_LAWYER, ACCESS_LIBRARY, ACCESS_MAINT_TUNNELS,
+				ACCESS_MEDICAL, ACCESS_MECH_ENGINE, ACCESS_MECH_MEDICAL, ACCESS_MECH_SCIENCE, ACCESS_MECH_SECURITY,
+				ACCESS_MINING_STATION, ACCESS_MINERAL_STOREROOM, ACCESS_MORGUE, ACCESS_PSYCHOLOGY, ACCESS_RC_ANNOUNCE,
+				ACCESS_RESEARCH, ACCESS_SECURITY, ACCESS_TELEPORTER, ACCESS_THEATRE, ACCESS_VAULT, ACCESS_WEAPONS
+				)
+//Removes Crematorium access, adds access to the Service hall
```

Comment:
I'm not sure how to feel about this- perhaps these should instead modify the list in New()